examples/asa/README.md

```diff
@@ -12,11 +12,21 @@ These scripts contain logic to transfer assets to `john-account` but other accou
 
 Balances can be queried by executing `algob run scripts/query/john-balances.js`.
 
-This example also includes creation of ASA owned by contract with following features:
-- Maximum number of ASA creation limit is 1.
-- Transaction must be signed by owner to get ASA out from contract
-- Only owner can call - update smart contract to set a new owner
-- Only owner can transfer ASA out from the smart contract and maximum payment is 100 ASA.
+This example also includes stateless smart contracts in (`assets/` directory) that showcase the two different modes of operation (contract & signature delegation):
+- [`2-gold-contract-asc.teal`](https://github.com/scale-it/algo-builder/blob/master/examples/asa/assets/teal/2-gold-contract-asc.teal): A lsig in contract mode which approves the transaction if Transaction type is pay or axfer, algo amount AND asset amount is <= 100.
```

Comment:
Could you please add the conditions/validations in bullet points? I think that will be more clear.

Reply:
Good idea